latest

```diff
@@ -0,0 +1 @@
+1.2.0
```

Comment:
Where is it used? Shouldn't it make `/latest` show the plain text instead of redirect?

Reply:
This is a way how GitHub render symlinks. `latest` just a regular symlink to latest release folder, `1.2.0` for now.